### ISSUE TITLE
Feature request: Possible to run cronjob via web

### ISSUE THREAD

**Issue by kristianrl** [phpservermon/phpservermon#215]
I made it possible to execute the cronjob by removing the following from `cron/status.cron.php` 

```
if(!psm_is_cli()) {
    die('This script can only be run from the command line.');
}
```

It seems to run perfectly fine.

I do understand the reasons not to have this publicly accessible, but ask that you add a possibility to enable/disable this (and just leave it disabled as default).

**Comment by sadortun** [comment]
Good idea, we are accepting PR.

**Comment by Abawell** [comment]
I dont really understand why you need that. You have the Update button that does the same thing.

**Comment by sadortun** [comment]
To be Able to run "http cron" on share hosting servers

**Comment by ZaDarkSide** [comment]
:+1:

**Comment by nixis** [comment]
Hi,
my workaround:

Create a file execute.php (or something else) in the cron folder:
`nano execute.php`

and fill it with:
```php
<?php
exec('php /FOLDERS/TO/phpservermon/cron/status.cron.php');
?>
```
now execute 

`http://yourdomain/cron/execute.php`

**Comment by Jonchun** [comment]
> Create a file execute.php (or something else) in the cron folder:

If the purpose of this was to run a cron over http on shared hosting servers, this wouldn't help because 99% of shared hosting servers will have exec() disabled, and if it doesn't, you have bigger problems...

**Comment by nixis** [comment]
@Jonchun If exec() is enabled it will work, my shared host provider has exec() enabled...

**Comment by pieter-groeneweg** [comment]
#358 may very well be what you are looking for.

**Comment by sadortun** [comment]
Resolved by #358 thanks @pieter-groeneweg
